examples/01_Exact_GPs/Simple_GP_Regression.ipynb

```diff
@@ -96,14 +96,21 @@
     "This specification leaves a large amount of flexibility when defining a model. For example, to compose two kernels via addition, you can either add the kernel modules directly:\n",
     "\n",
     "```python\n",
-    "self.covar_module = ScaleKernel(RBFKernel() + WhiteNoiseKernel())\n",
+    "self.covar_module = ScaleKernel(RBFKernel() + LinearKernel())\n",
```

Comment:
Does this still produce a reasonable plot with a linear kernel?

Reply:
Yeah it still looks fine